spec/factories/users.rb

```diff
@@ -4,5 +4,13 @@
     sequence(:last_name) { |n| "Elliot#{n}" }
     sequence(:email) { |n| "faithful#{n}@bath.uk" }
     sequence(:password) { |n| "fredrick#{n}" }
+
+    factory :admin_user, aliases: [:admin] do
+      sequence(:first_name) { |n| "Anne#{n}" }
+      sequence(:last_name) { |n| "Elliot#{n}" }
+      sequence(:email) { |n| "faithful#{n}@bath.uk" }
+      sequence(:password) { |n| "fredrick#{n}" }
+      admin true
+    end
   end
```

Comment:
For child factories like this, it may not be necessary to care about the creation of all attributes, just the ones that need to change (admin true).
```
factory :admin_user, aliases: [:admin] do
  admin true
end
```

 So if you were to call the admin_user factory, it would define the rest of attributes based on the parent factory specification.